data/mp/eda_mp_trj.py

```diff
@@ -138,10 +156,10 @@ def tile_count_anno(hist_vals: list[Any]) -> dict[str, Any]:
     symbol_pos=(0.2, 0.8),
     log=True,
     cbar_title="Magmoms ($μ_B$)",
-    cbar_title_kwds=dict(fontsize=16),
+    cbar_title_kwargs=dict(fontsize=16),
     cbar_coords=(0.18, 0.85, 0.42, 0.02),
     # annotate each element with its number of magmoms in MPtrj
-    anno_kwds=tile_count_anno,
+    # anno_kwds=tile_count_anno,
```

Comment:
does this have a replacement?

Reply:
tracked in https://github.com/janosh/pymatviz/issues/199